crates/db_schema/src/impls/local_user.rs

```diff
@@ -295,6 +299,26 @@ impl LocalUser {
   }
 }
 
+/// Checks to see if you can mod an item.
+///
+/// Caveat: Since admin status isn't federated or ordered, it can't know whether
+/// item creator is a federated admin, or a higher admin.
+/// The back-end will reject an action for admin that is higher via
+/// LocalUser::is_higher_mod_or_admin_check
+#[diesel::dsl::auto_type]
+pub fn local_user_can_mod() -> _ {
```

Comment:
Same goes for this select. It's used in several places like the CommentView and PostView.